DSA-Campaign-Uplift-Estimation/src/main/webapp/Create/create.html

```diff
@@ -165,7 +165,7 @@ <h3>Location 1</h3>
 
                 <div id="negativeLocations" style="display: block;">
                   <div class="form-group" id='nlocation1'>
-                    <h3>Negative Location 1</h3>
+                    <h3>Negative Location</h3>
```

Comment:
Negative Locations instead of Negative Location

Reply:
Fixed